skyportal/models/localization.py

```diff
@@ -141,3 +146,31 @@ def flat(self):
             return healpy.reorder(result, 'NESTED', 'RING')
         else:
             return (self.flat_2d,)
+
+    tiles = relationship("LocalizationTile")
+
+
+class LocalizationTile(Base):
+    """This is a single tile within a skymap (as in the Localization table).
+    Each tile has an associated healpix id and probability density."""
+
+    localization_id = sa.Column(
+        sa.ForeignKey('localizations.id', ondelete="CASCADE"),
+        nullable=False,
```

Comment:
```suggestion
        primary_key=True,
```

Otherwise Postgres would complain if you had two different sky maps that contained the same HEALPix tile.```